<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Joris Nonnast</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="title">
         <string>Control</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="comapreMethodsPushButton">
           <property name="toolTip">
            <string>Finds tirangles in all six samples and shows the time meassured side by side+#</string>
           </property>
           <property name="text">
            <string>Compare Methods</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string/>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QRadioButton" name="radioButton">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="toolTip">
                 <string>Finds triangles based on contour, approximate a polygon for each contour and show all three sided polygons</string>
                </property>
                <property name="text">
                 <string>Method 1</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButton_2">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="toolTip">
                 <string>Finds triangles based on contour and blob analysis, filters all found blobs by shapefactor (area / diameter^2)</string>
                </property>
                <property name="text">
                 <string>Method 2</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="adjustCannyThresholdLabel">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Adjust Canny threshold</string>
                </property>
                <property name="scaledContents">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QSlider" name="adjustCannyUpperHorizontalSlider">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="whatsThis">
                   <string>upper threshold</string>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="adjustCannyUpperSpinbox"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QSlider" name="adjustCannyLowerHorizontalSlider">
                  <property name="whatsThis">
                   <string>lower threshold</string>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="adjustCannyLowerSpinbox"/>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QPushButton" name="findTrianglesPushButton">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="toolTip">
                 <string>Finds traingles in selected picture and marks them with a green outline</string>
                </property>
                <property name="text">
                 <string>Find Triangles</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="showStepsCheckBox">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>show steps</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="DropEnabledGraphicsView" name="imageGraphicsView">
        <property name="acceptDrops">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>PIck a sample or drag &amp; drop an image file (.jpg, .png, .bmp)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QComboBox" name="pickSampleComboBox">
      <property name="currentText">
       <string>Sample 1</string>
      </property>
      <item>
       <property name="text">
        <string>Sample 1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sample 2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sample 3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sample 4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sample 5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sample 6</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DropEnabledGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>custom_widget/dropenabledgraphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
